webapp/accounts/views.py

```diff
@@ -188,3 +187,7 @@ def form_valid(self, form):
         messages.add_message(self.request, messages.SUCCESS,
                              "Je profiel is aangepast.")
         return super(EditProfileView, self).form_valid(form)
+
+class MemberList(LoginRequiredMixin, ListView):
+    model = VokoUser
```

Comment:
I just realized, this will query all users including inactive (non activated) users. I think this statement can be replaced by a `queryset` statement like `queryset = VokoUser.objects.filter(is_active=True)`